internal/stoppropaganda/http.go

```diff
@@ -16,18 +16,21 @@ func fasthttpRequestHandler(ctx *fasthttp.RequestCtx) {
 }
 
 type StatusStruct struct {
-	DNS      map[string]*DNSServer `json:"DNS"`
-	Websites map[string]*Website   `json:"Websites"`
+	DNS      map[string]*DNSServerStatus `json:"DNS"`
+	Websites map[string]*WebsiteStatus   `json:"Websites"`
+}
+type StatusService struct {
```

Comment:
Why did you create `StatusService`? In order to wrap between brackets like this?
```json
{
    "DNS": {
        "..."
    },
    "Websites": {
        "..."
    }
{
```

Reply:
Because after i removed pointer `mux *sync.Mutex` there was a warning on `json.MarshalIndent(statusStruct)`.
`call of json.MarshalIndent copies lock value: StatusStruct contains sync.Mutex`

Maybe mutex pointer was better then...

Reply:
Also, separation of data and business logic is a good practice :yum:

Reply:
This was the reason why I used mutex pointers, so I can copy a structure with mutexes ;) Your approach is also fine.